src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -134,6 +133,8 @@ export default class WaypointModel {
          */
         this.isHold = false;
 
+        // TODO: This should be moved to the StandardRouteWaypointModel
```

Comment:
yeah, I'm not a huge fan of that pattern.  it works as a way to get the same object from two different classes.  I'm not sold on the idea that inheritance is the correct pattern here, though, either.